servicetalk-client-api/src/main/java/io/servicetalk/client/api/ConnectionFactory.java

```diff
@@ -31,7 +34,9 @@
      * Creates and asynchronously returns a connection.
      *
      * @param address to connect.
+     * @param observer {@link TransportObserver} that provides visibility into transport events associated with a new
```

Comment:
Instead of repeating the `TransportObserver` definition, just say `{@link TransportObserver} for the newly created connections`?